joe/joe.py

```diff
@@ -105,9 +105,9 @@ def main():
     arguments = docopt(__doc__, version=__version__)
 
     if (arguments['ls'] or arguments['list']):
-        _print_filenames()
+        print _print_filenames()
```

Comment:
printing a print function doesn't sound right in my book.
You can rename the function altogether.